### PR TITLE
add tests to sessions statistics service

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/app/sessions/services/sessions-statistics-service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    });
+
+    describe('calculateAvailableDuration', () => {
+        it('should give correct value when available', function () {
+            expect(sessionsStatisticsService.calculateAvailableDuration(moment.duration('PT1H'), moment.duration('PT30M')))
+                .toEqual(30);
+        });
+    });
+
+    describe('calculateAvailableDuration', () => {
```
Comment (zielinskikamil) [inline src/app/sessions/services/sessions-statistics-service.spec.ts]: you don't have to wrap each `it` that tests `calculateAvailableDuration` is separate `describe`. Just make one `desc` for them.

File: src/app/sessions/services/sessions-statistics-service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                hearingParts: [] as [HearingPart],
+                jurisdiction: undefined
+            };
+            expect(sessionsStatisticsService.calculateAllocatedHearingsDuration(session))
+                .toEqual(moment.duration(0));
+        });
+    });
+
+    describe('calculateUtilizedDuration', () => {
+        it('should calculate 50% of utilization', function () {
```
Comment (zielinskikamil) [inline src/app/sessions/services/sessions-statistics-service.spec.ts]: please use arrow functions

File: src/app/sessions/services/sessions-statistics-service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+let session: SessionViewModel;
+
+describe('SessionsStatisticsService', () => {
+    beforeAll(() => {
+        sessionsStatisticsService = new SessionsStatisticsService();
+    });
+
+    describe('calculateAllocatedHearingsDuration', () => {
+        it('should give one hearing part duration given session with one hearing', () => {
+            session = {
```
Comment (radoslawlandowskikainos) [inline src/app/sessions/services/sessions-statistics-service.spec.ts]: Can we create even a private method in this test getEmptySession that returns a session with all fields as undefined and then use the spread operator to fill only the required fields?

let session = { ...this.getEmptySession(), duration: 10, hearingParts: {[...this.getEmptyHearingPart(), duration: moment.duration(PT20M)}]}

Comment (radoslawlandowskikainos) [reply src/app/sessions/services/sessions-statistics-service.spec.ts]: Or even go for a separate Builders

**Pull request merged**